### PR TITLE
APIGOV-20072 - update offline usage reporting documentation

### PR DESCRIPTION
**Pull request by jcollins-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

update offline usage reporting documentation

## Deploy preview link

- https://deploy-preview-2054--axway-open-docs.netlify.app/docs/central/connected_agent_common_reference/traceability_usage/
- https://deploy-preview-2054--axway-open-docs.netlify.app/docs/central/connect-api-manager/agent-variables/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@kflan-io @dfeldick @sbolosan @tsjohns9 @sagarkal - would some of you please validate the changes.

File: content/en/docs/central/connected_agent_common_reference/traceability_usage.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ If the Traceability Agent is stopped while there are still remaining usage event
 
 When offline usage reporting is on, `CENTRAL_USAGEREPORTING_OFFLINE=true` (default = false), the `CENTRAL_USAGEREPORTING_SCHEDULE` variable determines how often usage numbers are saved (default and minimum = "@hourly"). For additional cron schedules information, see [Scheduled jobs](https://github.com/Axway/agent-sdk/blob/main/pkg/jobs/README.md#scheduled-jobs). Note that offline ignores the `CENTRAL_USAGEREPORTING_INTERVAL` value that is only used for online reporting.
 
+In addition to setting `CENTRAL_USAGEREPORTING_OFFLINE=true` the `CENTRAL_ENVIRONMENTID` variable will have to be set.  The usage report requires a valid Environment ID and in not able to retrieve it in Offline mode.  This can be retrieved with the axway cli see [Retrieve a list of all available environments](/docs/central/cli_central/cli_environments#retrieve-a-list-of-all-available-environments).
```
Comment (kflan-io) [inline content/en/docs/central/connected_agent_common_reference/traceability_usage.md:126]: Maybe add a comma for consistency:
...axway cli**,** see

**Review by kflan-io — APPROVED** [review]

**Review by dfeldick — APPROVED** [review]

File: content/en/docs/central/connected_agent_common_reference/traceability_usage.md (hunk 1 above)
Comment (jcollins-axway) [reply content/en/docs/central/connected_agent_common_reference/traceability_usage.md]: done, thanks

**Review by sbolosan — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Pull request merged**